docs: Component Props - edit pass (#830)
BuilderIO/qwik: packages/docs/src/pages/tutorial/props/basic/index.mdx

layout: tutorial
---

Web applications are built up from components in the same way that general applications are built up from functions.

> Composing functions would not be very useful you couldn't pass in parameters. In the same way that functions have parameters, components have "props". A component uses props to pass data from parent to child components.

Reviewer: missing an "if" after "be very useful"

Plus - I would modiy the last sentence to "A component uses props to pass data to its children components"

Author: thanks for catching that 😄

> Composing functions would not be very useful you couldn't pass in parameters. In the same way that functions have parameters, components have "props". A component uses props to pass data from parent to child components.

Author: ```suggestion
Composing functions would not be very useful if you couldn't pass in parameters. In the same way that functions have parameters, components have "props". A component uses props to pass data its children components.
```
